guides/v2.2/config-guide/mq/manage-message-queues.md

```diff
@@ -36,7 +36,7 @@ You can also use a process manager such as [Supervisor](http://supervisord.org/i
 
 #### Behavior by default
 
-* Cron job `consumers_runner` is enabled
+* Cron job `consumers_runner` is disabled
 * Cron job `consumers_runner` runs all defined consumers
 * Each consumer processes 1000 messages and then terminates
```

Comment:
You may update this line though, the default number of messages is 10000, not 1000.